packages/features/settings/BookerLayoutSelector.tsx

```diff
@@ -85,19 +85,28 @@ const BookerLayoutFields = ({ settings, onChange, showUserSettings }: BookerLayo
   // Converts the settings array into a boolean object, which can be used as form values.
   const toggleValues: BookerLayoutState = bookerLayoutOptions.reduce((layouts, layout) => {
     layouts[layout] = !shownSettings?.enabledLayouts
-      ? true
+      ? defaultBookerLayoutSettings.enabledLayouts.indexOf(layout) > -1
       : shownSettings.enabledLayouts.indexOf(layout) > -1;
     return layouts;
   }, {} as BookerLayoutState);
 
   const onLayoutToggleChange = useCallback(
     (changedLayout: BookerLayouts, checked: boolean) => {
+      const newEnabledLayouts = Object.keys(toggleValues).filter((layout) => {
+        if (changedLayout === layout) return checked === true;
+        return toggleValues[layout as BookerLayouts] === true;
+      }) as BookerLayouts[];
+
+      const isDefaultLayoutToggledOff = newEnabledLayouts.indexOf(defaultLayout) === -1;
+      const firstEnabledLayout = newEnabledLayouts[0];
+
       onChange({
-        enabledLayouts: Object.keys(toggleValues).filter((layout) => {
-          if (changedLayout === layout) return checked === true;
-          return toggleValues[layout as BookerLayouts] === true;
-        }) as BookerLayouts[],
-        defaultLayout,
+        enabledLayouts: newEnabledLayouts,
+        // If default layout is toggled off, we set the default layout to the first enabled layout
+        // if there's none enabled, we set it to month view.
+        defaultLayout: isDefaultLayoutToggledOff
```

Comment:
Added behaviour that if you disable the layout that's currently marked as the default, that we switch to the next best candidate. Otherwise you would have a layout marked as default that's actually a disabled radio button.